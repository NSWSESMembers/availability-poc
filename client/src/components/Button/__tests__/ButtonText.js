import React from 'react';
import renderer from 'react-test-renderer';

import ButtonText from '../ButtonText';

test('renders correctly', () => {
  const tree1 = renderer.create(
    <ButtonText
      title="title"
      description="description"
      icon
    />,
  ).toJSON();
  expect(tree1).toMatchSnapshot();
});
