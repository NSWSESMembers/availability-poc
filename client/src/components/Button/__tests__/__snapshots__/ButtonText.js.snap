// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "justifyContent": "space-between",
      "paddingBottom": 5,
      "paddingTop": 5,
    }
  }
>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "fontSize": 11,
          "fontWeight": "900",
          "paddingTop": 5,
        }
      }
    >
      TITLE
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "paddingBottom": 5,
        }
      }
    >
      description
    </Text>
  </View>
  <Text
    accessible={true}
    allowFontScaling={false}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "#EEE",
          "fontSize": 40,
        },
        undefined,
        Object {
          "fontFamily": "FontAwesome",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
      ]
    }
  >
    ï„…
  </Text>
</View>
`;
